src/System Application/App/Pdf/README.md

```diff
@@ -0,0 +1,80 @@
+This module provides an internal API for working with PDF documents in Business Central. It supports basic PDF manipulation tasks, primarily focused on extracting embedded XML e-invoices and metadata from PDFs, such as Factur-X, XRechnung, and ZUGFeRD formats.
+
+You can use this module to:
+
+- Extract invoice attachments from PDF documents
+
+- Download all attachments as a ZIP archive
+
+- List the names of all embedded attachments
+
+- Retrieve PDF metadata (author, title, page size, etc.)
+
+- Number of pages 
+
+### Extract an embedded invoice from a PDF
+```
+procedure Example(PdfStream: InStream)
+var
+    TempBlob: Codeunit "Temp Blob";
+    Success: Boolean;
+    PDFDocumentImpl: Codeunit "PDF Document Impl.";
```

Comment:
All README code samples should call PDFDocument instead of PDFDocumentImpl.

For the "Read PDF metadata" section, let's use another example than page count, as the latter is already handled in section "Get number of pages".